### ISSUE TITLE
Android support

### ISSUE THREAD

**Issue by RobertZaks** [XapaJIaMnu/translateLocally#149]
Are you planning to support translateLocally on android in the future?

**Comment by XapaJIaMnu** [comment]
Yes, it is planned! There's a non public alpha where the QT project is compiled to android, but the process is very ad-hoc for now. We plan to release beta android/iphone support using the direct QT compilation.
